install-hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/bash
+
+cp ./hooks/* ./.git/hooks/
```

Comment:
I would prefer this as an instruction in the README.md

Reply:
@Xazin is this good to add to the README file?

## Pre-push checks

We have set up Git hooks to prevent extra whitespace and other possible mistakes before pushing the code to your branch. Run the below command in the project root directory to copy the pre-push hooks to your local `.git` directory.
```bash
  cp ./hooks/* ./.git/hooks/
```

Reply:
Looks good

Reply:
Sounds good